meilisearch/src/routes/multi_search.rs

```diff
@@ -112,6 +112,7 @@ pub async fn multi_search_with_post(
                         ));
                     }
 
+                    let features = index_scheduler.features();
```

Comment:
I don't think we should be getting the features another time here (they could have changed!).

Rather, we should reuse the top-level `features` binding.

Reply:
Oops yes I missed it